assets/app/scripts/services/builds.js

```diff
@@ -73,7 +74,8 @@ angular.module("openshiftConsole")
             $scope.alerts["rebuild"] = 
             {
               type: "success",
-              message: "Build " + buildName + " is being rebuilt as " + build.metadata.name + "."
+              message: "Build " + buildName + " is being rebuilt as " + build.metadata.name + ".",
+              link: "project/" + build.metadata.namespace + "/browse/builds/" + buildConfigName +"/" + build.metadata.name
```

Comment:
use the navigateResourceURL filter to generate this link. we are still migrating all the old places that were manually building links but anything new should use this.